README.md

```diff
@@ -13,6 +13,28 @@
   <img src="Screenshot.png" alt="Screenshot" height="600">
 </p>
 
+# About
+
+The performance controller for [Apple Silicon](https://en.wikipedia.org/wiki/Apple-designed_processors) is tasked with balancing the competing interests of performance and energy efficiency. For light workloads, the performance controller saves energy by scheduling work onto efficiency cores and running them at low clock speeds. For heavier workloads, it moves the work to performance cores and ramps up their clock speeds.
+
+For most apps, the performance controller strikes a good balance between these interests. Low-latency pro audio apps, however, are especially demanding, and performance isn't always optimal by default. AudioPerfLab provides an environment for testing workarounds to improve multi-threaded audio performance.
+
+To simulate a demanding DSP workload, the app synthesizes a unison saw wave chord using additive synthesis. Sine wave partials are computed in parallel using a pool of real-time threads. Increasing the number of sine waves increases the DSP load. Pressing &nbsp;<sub><img src="Icons/Play.png" alt="Play" height="16pt"></sub>&nbsp; increases the number of waves all at once to test the system's response to a sudden load increase.
+
+The [cores visualization](#cores) shows how audio threads are scheduled onto cores over time and the [energy visualization](#energy) shows power consumption. The main two workarounds to avoid CPU throttling are adding [artificial load to real-time threads](#minimum-load) and running [busy threads](#busy-threads) that perform low-energy work in the background. There are additional controls to tweak things like the [number of audio threads](#process-threads) and if those threads [join a workgroup](#work-interval).
+
+See [High Performance Audio on iOS](https://youtu.be/ywrLAv5WNq4) from ADC '19 for more information.
+
+# Recommendations
+
+On iOS 13, real-time multi-threading doesn't work at low buffer sizes (< 512). For these buffer sizes, worker threads get scheduled onto efficiency cores with low clock speeds. These threads run so slow that a task that could normally (on a fast core) be processed in 0.5ms ends up taking more than 3ms, overshooting the 2.7ms deadline for a buffer size of 128.
```

Comment:
Nit: "a task that could be processed in 0.5ms on a fast core ends up..." might read a bit better, the "normally" with a parenthetical is a bit awkward.

Reply:
Good point, changed!